egs/wsj/s5/steps/nnet2/get_egs_discriminative2.sh

```diff
@@ -256,39 +255,23 @@ if [ $stage -le 10 ]; then
 priors_egs_list=
 for y in `seq $num_archives_priors`; do
```

Comment:
Why is num_archives_priors still being used ? I thought the entire point was to just have one job for prior computation.